### PR TITLE
Remove dependency on aspnetcore-ci-dev NuGet feed

### PR DESCRIPTION
**Pull request by natemcmaster**
Solves the 🐔 and 🍳 problem of building BuildTools if aspnetcore-ci-dev were to be cleared.

This required duplicating Microsoft.Extensions.CommandLineUtils.Sources into this repo.

cc @mikeharder - build from source

### CODE REVIEW

File: modules/BuildTools.Tasks/BuildTools.Tasks.csproj [hunk 1]
```diff
@@ -9,16 +9,15 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\..\shared\Microsoft.Extensions.CommandLineUtils.Sources\**\*.cs" />
```
Comment (pranavkm) [inline modules/BuildTools.Tasks/BuildTools.Tasks.csproj]: The 2.0.0 version of this package exists on aspnetcore-master (which isn't afflicted by the rooster and scrambled omelette problem).

Comment (natemcmaster) [inline modules/BuildTools.Tasks/BuildTools.Tasks.csproj]: We'd still run into this in the future: https://github.com/aspnet/Coherence-Signed/issues/558

Comment (pranavkm) [inline modules/BuildTools.Tasks/BuildTools.Tasks.csproj]: Isn't that solved the same way we'd solve all our other 3rd party dependency issues?

Comment (natemcmaster) [inline modules/BuildTools.Tasks/BuildTools.Tasks.csproj]: Not exactly. 3rd party deps are on NuGet.org, but Ms.Ext.CommandLineUtils.Sources won't be.

Comment (pranavkm) [inline modules/BuildTools.Tasks/BuildTools.Tasks.csproj]: Why does the feed location make a difference? I'm not trying to disavow checking in sources, I"m just trying to figure out how any of this is meant to work.

Comment (natemcmaster) [inline modules/BuildTools.Tasks/BuildTools.Tasks.csproj]: The hypothetical scenario is that there is no feed at all. If we don't have any dotnet.myget.org feed available, we're stuck.

FWIW I wish we didn't have to copy source either, but it seems better than the alternative: git submodules.

**Pull request merged**